src/fastoad/notebooks/02_explanation/05_FAST-OAD.ipynb

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
<blockquote>it should be <a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/91dd1f0886da2c366c84b520184b5d61072761c5/src/fastoad/notebooks/02_explanation/working_folder/MDA_global_group/problem_outputs.xml" target="_blank">here</a></blockquote><p>it should be <a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/91dd1f0886da2c366c84b520184b5d61072761c5/src/fastoad/notebooks/02_explanation/working_folder/MDA_global_group/problem_outputs.xml" target="_blank">here</a> (if not, then something went wrong when running <code>evaluate_problem ).</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Changed in dfe233f7